docker/Dockerfile

```diff
@@ -1,41 +1,29 @@
-FROM registry.service.dsd.io/ruby:2.6.3
+FROM ruby:2.6.3-alpine3.8
 
-# add official nodejs repo
-RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
-        echo 'deb https://deb.nodesource.com/node jessie main' > /etc/apt/sources.list.d/nodesource.list
+# Install pre-requisites for building nokogiri & pg gems, as well
+# as the packages for converting and generating PDF's
+RUN apk --update add --virtual build_deps \
```

Comment:
For discussion, be good  (for me if no one else, to understand what these packages are for. In my dummy app i have:
```
# Additional packages/libraries
# - install regular stuff build-base
# - install libxml2-dev libxslt-dev required for nokogiri
# - install postgresql-dev required for postgres
# - install nodejs needed for ExecJS functionality
# - install yarn needed for asset compilation
#
```

I know what pdftk, redis and libreoffice are for, but not clear what ruby-dev, libc-dev, linux-headers, runit and git are for

Reply:
The `--virtual <name>` flag seems to be for when you want to clear up dependences later. [found this explanation](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#virtual-packages)  - since we are not cleaning `build_deps` up we may not need it although it is possible we could clear up the `-dev` packages as they may only bre required for the build